_posts/2016-xx-xx-release-0.16.md

```diff
@@ -119,6 +119,116 @@ If you are using the low-level qcprot algorithm your self intead of our provided
+
+```python
+hes_similarities, details = encore.hes([u1, u2])
+print hes_similarities
+```
+```
+[[        0.         38279683.9587939]
+ [ 38279683.9587939         0.       ]]
+```
+
+2) the Clustering Ensemble Similarity measure:
+
+```python
+ces_similarities, details = encore.ces([u1, u2])
+print ces_similarities
+```
+```
+[[ 0.          0.68070702]
+ [ 0.68070702  0.        ]]
+```
+
+or 3) the Dimensionality Reduction Ensemble Similarity measure:
+
+```python
+dres_similarities, details = encore.dres([u1, u2])
+print dres_similarities
+```
+```
+[[ 0.          0.65434461]
+ [ 0.65434461  0.        ]]
+```
+Similarities are written in a square symmetric matrix having the same dimensions and ordering as the input list, with each element being the similarity value for a pair of the input ensembles. 
+
+The encore library includes a general interface to various clustering and dimensionality reduction algorithms (through the scikit-learn package), which makes it easy to switch between clustering and dimensionality reduction algorithms when using the `ces` and `dres` functions. The clustering and dimensionality reduction functionality is also directly available through the `cluster` and `reduce_dimensionality` functions. For instance, to cluster the conformations from the two universes defined above, we can write:
+```python
+cluster_collection = encore.cluster([u1,u2])
+print cluster_collection
+```
+```
+0 (size:5,centroid:1): array([ 0,  1,  2,  3, 98])
+1 (size:5,centroid:6): array([4, 5, 6, 7, 8])
+2 (size:7,centroid:12): array([ 9, 10, 11, 12, 13, 14, 15])
+…
+```
+In addition to standard cluster membership information, the `cluster_collection` output keep track of the origin of each conformation, so you check how the different trajectories are represented in each cluster:
+```python
+print [cluster.metadata["ensemble_membership"] for cluster in cluster_collection]
+```
+```
+[array([1, 1, 1, 1, 2]), array([1, 1, 1, 1, 1]), array([1, 1, 1, 1, 1, 1, 1]), array([1, 1, 1, 1, 1, 1]), array([1, 1, 1, 1, 1, 1]), array([1, 1, 1, 1, 1, 1]), array([1, 1, 1, 1]), array([1, 1, 1, 1, 1, 1]), array([1, 1, 1, 1]), array([1, 1, 1, 1, 1]), array([1, 1, 1, 1]), array([1, 1, 1, 1, 1, 1]), array([1, 1, 1, 1, 1, 1]), array([1, 1, 1, 1, 1, 1, 1, 1]), array([1, 1, 1, 1, 1, 1, 1]), array([1, 1, 1, 1, 1, 1, 1]), array([1, 1, 1, 1, 1, 1, 1]), array([2, 2, 2, 2, 2]), array([2, 2, 2, 2, 2, 2]), array([2, 2, 2, 2, 2, 2]), array([2, 2, 2, 2, 2]), array([2, 2, 2, 2, 2]), array([2, 2, 2, 2]), array([2, 2, 2, 2, 2]), array([2, 2, 2, 2, 2, 2]), array([2, 2, 2, 2]), array([2, 2, 2, 2, 2, 2]), array([2, 2, 2, 2]), array([2, 2, 2, 2, 2]), array([2, 2, 2, 2, 2, 2]), array([2, 2, 2, 2, 2]), array([2, 2, 2, 2, 2, 2]), array([2, 2, 2, 2, 2, 2, 2, 2, 2]), array([2, 2, 2, 2, 2, 2, 2]), array([2, 2, 2, 2, 2, 2, 2])]
+```
+
+For further details, see the documentation of the individual functions within Encore.
```

Comment:
Done. I've moved this code to the docs for the `cluster.py` module. Will push the changes to the docs themselves in a few days (need to go over them all again as part of [ MDAnalysis/mdanalysis#1047](https://github.com/MDAnalysis/mdanalysis/issues/1047))